### ISSUE TITLE
Unable to build iOS App after changing Linker Settings

### ISSUE THREAD

**Issue by sharathm** [dkornev/TwilioVoiceXamarinIOS#7]
Hi,
We recently used the binding to integrate voice calling into the iOS Application, and apple is rejecting the build saying we are referencing HealthKit, and so we found that we had to skip Healthkit as a dependency and so gave `--linkskip=HealthKit` to make a build in mtouch arguments, but this is causing a problem with Twilio because we get the following error now during build time. 

```
Error MT2101: Can't resolve the reference 'Foundation.NSArray 
Foundation.NSArray::FromNSObjects(!!0[])', referenced from the method 'System.Void 
Twilio.Voice.iOS.TVOCallOptionsBuilder::set_PreferredAudioCodecs(Twilio.Voice.iOS.TVOAudio
Codec[])' in 'Xamarin.iOS, Version=0.0.0.0, Culture=neutral, 
PublicKeyToken=84e04ff9cfb79065'. (MT2101) 
```

Can you please help us solve this?

Details:
Xamarin iOS SDK - 12.4.0.64
Twilio SDK - 5.31.1
Twilio  Voice Xamarin iOS Bindings - 3.1.0.1

**Comment by sharathm** [comment]
Updated:
Moved to the latest Twilio SDK and Twilio Voice Xamarin iOS Bindings, but the issue still persists.

**Comment by dkornev** [comment]
Hi @sharathm ,
I don't see that _Twilio.Voice_ binding is using _HealthKit_ anywhere. According to https://github.com/dkornev/TwilioVoiceXamarinIOS/blob/master/src/libTwilioVoice.linkwith.cs it doesn't use it.

So, I'm pretty sure that it's not a binding issue and doesn't look like a Twilio issue too for me.
Please check your project settings and make sure that you set _--registrar:static_  as additional mtouch argements

**Comment by sharathm** [comment]
@dkornev i agree, this library has nothing to do with HealthKit, but I was just trying to tell the situation that led to the error above. Whenever we set the linker options to anything other than `Don't Link` this error shows up, hence posted the issue here.

**Comment by dkornev** [comment]
@sharathm I'm not sure that it helps, but try to add --registrar:static -cxx -gcc_flags -dead_strip flags as additional mtouch arguments. It fixed the other issue (#10) and may also help you
